### PR TITLE
Footer FED

### PR DESCRIPTION
**Pull request by liamjohnston**
TODO, change base branch once nav merged in (which is awaiting a BED PR to be merged).

The TODOs will be resolved in separate PRs later.

cdh dev springload nz_ (1)
cdh dev springload nz_ (2)
Screenshot 2024-05-16 at 1 37 15 PM

### CODE REVIEW

File: cdhweb/static_src/global/components/footer.scss [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+  flex-direction: column;
+  gap: var(--v-link-space);
+
+  :where(p) {
+    margin: 0;
+  }
+}
+
+.footer__contact-links {
+}
```
Comment (wilsonespina) [inline cdhweb/static_src/global/components/footer.scss]: Is this needed?

File: templates/includes/search_form.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{% load core_tags %}
+
+{% comment %} 
+  This form will appear in more than one place on the page (header, footer, possibly elsewhere).
+  So `uniqueID` is used to prefix IDs (and their `for`s), so we don't get duplicate IDs in the markup.
+{% endcomment%}
```
Comment (wilsonespina) [inline templates/includes/search_form.html]: nitpick:
```suggestion
{% endcomment %}
```

File: templates/snippets/footer_menu.html [hunk 3]
```diff
@@ -1,26 +1,94 @@
...
-    {% endfor %}
+<footer class="footer">
 
-    {% if physical_address %}
-        <div>{{ physical_address.body|richtext }}
+  <div class="content-width">
+    
+    <div class="footer__top grid-standard">
+      <div class="footer__logo">
+        <a href="/" aria-label="The Center for Digital Humanities at Princeton">
```
Comment (wilsonespina) [inline templates/snippets/footer_menu.html:9]: I finally know what CDH stands for! 😂

**Review by wilsonespina — APPROVED** [review]
Looks good!

File: cdhweb/static_src/global/components/footer.scss (hunk 1 above)
Comment (liamjohnston) [reply cdhweb/static_src/global/components/footer.scss]: Nope, will remove. Ta.

**Pull request merged**